packages/hint-font-relative-units/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "ava": {
+    "failFast": false,
+    "files": [
+      "dist/tests/**/*.js"
+    ],
+    "timeout": "1m"
+  },
+  "description": "Ensure font styles use relative units to enable user stylesheet overrides",
+  "devDependencies": {
+    "@hint/utils-tests-helpers": "^5.0.11",
+    "@types/node": "^12.7.5",
+    "@typescript-eslint/eslint-plugin": "^1.13.0",
+    "@typescript-eslint/parser": "^1.12.0",
+    "ava": "^1.4.1",
```

Comment:
**Note to @webhintio/core:** Make sure all dependencies are up to date before merging. We've had a few issues in the past with this.